ERCS/erc-7836.md

```diff
@@ -230,38 +230,19 @@ const response = await provider.request({
 
 ## Rationale
 
-Many session key use cases involve submitting operations on users' behalves. One example is an application that automates a user's DEX trading. In these cases, the wallet may provide a "wallet server" URL for the app to submit JSON-RPC requests to outside of the context of a browser.
+These endpoints extend the interface established by EIP-5792, given the emergent needs of application developers.
 
-It is not in the scope of this EIP, but someone could define a `walletServer` [EIP-5792](./eip-5792.md) capability that declares a public wallet server URL.
+Adding new endpoints for this specific use case is simpler than adding additional options to the `wallet_sendCalls` endpoint.
```

Comment:
I'm not sure it's worth calling this out; I don't think this could even be feasible with wallet_sendCalls since that RPC implicitly requires a signing step as a part of handling it, whereas here we are proposing two RPCs that bookend the signing step